app/forms/questionnaire_form.py

```diff
@@ -411,7 +411,6 @@ def _get_calculation_total(
         return result
 
     def answers_all_valid(self, answer_id_list: Sequence[str]) -> bool:
-        # pylint: disable=no-member
         # wtforms Form parents are not discoverable in the 2.3.3 implementation
```

Comment:
theres inconsistency with whether the wtf forms being discoverable comment is removed or not, can delete the others too

Reply:
Removed all wtf form comments relating to this 👍